ignite/engine/engine.py

```diff
@@ -538,7 +539,7 @@ def call_interrupt():
         self.logger.info("interrupt signaled. Engine will interrupt the run after current iteration is finished.")
         self.should_interrupt = True
 
-    def terminate(self) -> None:
+    def terminate(self, skip_event_completed: bool = False) -> None:
```

Comment:
I like the name of the `self.skip_completed_after_termination`, let's call this flag in the same way:
```suggestion
    def terminate(self, skip_completed: bool = False) -> None:
```